cmd/nerdctl/run.go

```diff
@@ -123,7 +123,7 @@ func setCreateFlags(cmd *cobra.Command) {
 
 	// #region network flags
 	// network (net) is defined as StringSlice, not StringArray, to allow specifying "--network=cni1,cni2"
-	cmd.Flags().StringSlice("network", []string{netutil.DefaultNetworkName}, `Connect a container to a network ("bridge"|"host"|"none"|<CNI>)`)
+	cmd.Flags().StringSlice("network", []string{netutil.DefaultNetworkName}, `Connect a container to a network ("bridge"|"host"|"none"|"container:<container>"|<CNI>)`)
```

Comment:
Please update the corresponding line in the README.md too

Reply:
It should be also noted that the `container` has to be precreated but `nerdctl compose up` and the restart manager do not care that dependency

Reply:
Thanks for reviewing! and yes, will do all the docs and tests